test-suite/shared/src/test/require-jdk11/org/scalajs/testsuite/javalib/util/concurrent/FlowTest.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.util.concurrent
+
+import java.util.concurrent.Flow
+
+import org.junit.Test
+import org.junit.Assert._
+
+class FlowTest {
+  import FlowTest._
+
+  @Test def testProcessor(): Unit = {
+    val processor = makeProcessor[Int, String]()
+    processor.subscribe(makeSubscriber[String]())
+    processor.onSubscribe(makeSubscription())
+    processor.onNext(42)
+    processor.onError(new Exception)
+    processor.onComplete()
+  }
+
+  @Test def testPublisher(): Unit = {
+    val publisher = makePublisher[Int]()
+    publisher.subscribe(makeSubscriber[Int]())
+  }
+
+  @Test def testSubscriber(): Unit = {
+    val subscriber = makeSubscriber[Int]()
+    subscriber.onSubscribe(makeSubscription())
+    subscriber.onNext(42)
+    subscriber.onError(new Exception)
+    subscriber.onComplete()
+  }
+
+  @Test def testSubscription(): Unit = {
+    val subscription = makeSubscription()
+    subscription.request(42)
+    subscription.cancel()
+  }
+
+}
+
+object FlowTest {
+
+  def makeProcessor[T, R](): Flow.Processor[T, R] =
```

Comment:
```suggestion
  def makeProcessor[T, R](): Flow.Processor[T, R] = {
```
Same for all the `makeX` methods.

According to our coding style:
https://github.com/scala-js/scala-js/blob/main/CODINGSTYLE.md#braces